tests/test_importexport.py

```diff
@@ -2265,6 +2265,20 @@ def test_bench_with_same_io_name(self):
         self.check_io(pyrtl.Input, ["G1", "G2", "G3"])
         self.check_io(pyrtl.Output, ["tmp3", "G4"])
 
+class TestOutputVerilog(unittest.TestCase):
+    def test_custom_module_name(self):
+        import io, pyrtl
```

Comment:
This import shouldn't be necessary since these libraries are already imported on L1 and L7